common/source/docs/common-cameras-and-gimbals.rst

```diff
@@ -17,45 +17,56 @@ Gimbals
 
 ArduPilot supports both brushless direct drive gimbals (Tarot, SimpleBGC, SToRM32)
 that have their own self-stabilization controllers and the simpler servo-driven
-gimbals in which ArduPilot controls the stabilisation.
+gimbals in which ArduPilot controls the stabilisation. Some gimbals also integrate still or video cameras.
 
+-  :ref:`Servo Gimbals <common-camera-gimbal>` — older-style servo-driven gimbal where ArduPilot provides stabilisation
 -  :ref:`Brushless PWM <common-brushless-pwm-gimbal>` - brushless gimbals that accept PWM or SBUS input for angle control
+
+Specific OEM Gimbal models
+--------------------------
+
 -  :ref:`DJI RS2 and RS3-Pro gimbals <common-djirs2-gimbal>`
 -  :ref:`Gremsy Mio, Pixy, S1, T3, T7 and ZIO <common-gremsy-pixyu-gimbal>` - high quality 3-axis gimbals
--  :ref:`Servo Gimbals <common-camera-gimbal>` — older-style servo-driven gimbal where ArduPilot provides stabilisation
 -  :ref:`SimpleBGC (aka AlexMos) Gimbal Controller <common-simplebgc-gimbal>` - a popular 2-axis or 3-axis brushess gimbal controller which uses a custom serial interface
 -  :ref:`Siyi ZR10, ZR30 and A8 <common-siyi-zr10-gimbal>` - 3-axis gimbal and camera
 -  :ref:`SToRM32 Gimbal Controller <common-storm32-gimbal>` — an inexpensive 2-axis or 3-axis brushless gimbal controller which responds to MAVLink commands (a richer format than PWM) over a serial interface
 -  :ref:`ViewPro gimbals <common-viewpro-gimbal>`
 -  :ref:`Xacti gimbals <common-xacti-gimbal>`
 
-Mount control is covered on the :ref:`common-mount-targeting` page.
+Gimbals may be attached to retractable mounts to prevent ground contact or to reduce air resistant in flight. Mount control is covered on the :ref:`common-mount-targeting` page.
 
 .. _common-cameras-and-gimbals_camera_shutter_triggering:
 
 Cameras
 =======
 
-Camera manufacturers use their own mechanisms for remote control of the
-camera (including its shutter). The topics explain how to configure the
-camera shutter, and list a number of different approaches for converting
-the output signal into the form expected by your particular camera:
+Camera manufacturers use many differing mechanisms for remote control of the
+camera (including its shutter or video recording). The following topics explain how to configure camera shutter and controls, and list a number of different approaches for converting
+the output signal into the form expected by your particular camera. If the camera is integrated with the gimbal, control of the camera is detailed in that gimbals setup.
+
+General Camera Control and Parameters
+-------------------------------------
+
+-  :ref:`Camera General Parameters and Control<common-camera-controls>`
+-  :ref:`RunCam Camera Control <common-camera-runcam>`
+
+Shutter Control
```

Comment:
Here too I'm not a real fan of splitting up the groups.  I think we should just keep them in alphabetical order and list as many as we have.  Thanks for finding the missing items!

Reply:
I disagree on this....but I think I need to move the geotagger up into shutter control section....there is really only one camera here, the FLIR, otherwise this whole section is about camera shutter/triggers, not cameras....I would re-title Shutter Control  to Shutter/Trigger Controllers and move the geotagger into it since it includes trigger controller...the "Control of Specific Camera Models" would only have the FLIR and be retitled to "Cameras with Electronic Trigger Integrated"

Reply:
The RunCam is also a camera I think..

Reply:
yep, will move down...thanks